### PR TITLE
fix(injector): correct assign expression to argument 

### PR DESCRIPTION
**Pull request by azu**
This PR fixes to support following pattern:

```js
function fn(){ 
  if(condition) return statement;
  else return statement;
}
```

Previously, the conveted result is syntax error.

### CODE REVIEW

File: test/fixtures/if-else-no-block-return/output.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function fn() {
+    var sjsp__state = sjsp__start('example.js', 1, 15, 'fn', 'function fn() {');
+    if (true)
+        return function (arguments) {
+            var sjsp__return = 1;
+            sjsp__end(sjsp__state);
+            return sjsp__return;
+        }.call(this, arguments);
```
Comment (azu) [inline test/fixtures/if-else-no-block-return/output.js:8]: Previously, sjsp convert this to following:

```js
    if (true)
        return function (arguments) {
            var sjsp__return = 1;
            sjsp__end(sjsp__state);
            return sjsp__return;
        }.call(this, arguments);;
   else 
```

This is syntax error.

```js
if(true) 
  ;;
else
  ;;
// => SyntaxError: expected expression, got keyword 'else' 
```

File: package.json [hunk 2]
```diff
@@ -7,18 +7,19 @@
     "type": "git",
     "url": "git://github.com/45deg/node-sjsp.git"
   },
+  "main": "lib/injector.js",
```
Comment (azu) [inline package.json:10]: Add `main` field.
Useful for using it as library.

File: lib/injector.js [hunk 3]
```diff
@@ -101,10 +101,11 @@ function wrapReturn(returnStmt){
     );
 
     // rewrite __here__
-    wrapperFunc.expression.callee.object.body.body[0].declarations[0].init = 
+    wrapperFunc.expression.callee.object.body.body[0].declarations[0].init =
     returnStmt.argument;
 
-    returnStmt.argument = wrapperFunc;
+    // assign express to argument.
+    returnStmt.argument = wrapperFunc.expression;
```
Comment (azu) [inline lib/injector.js:108]: This is main fix.

**Comment by 45deg** [comment]
Thanks a lot!

**Pull request merged**